### PR TITLE
fix path lookup for pandoc etc. in electron builds

### PR DESCRIPTION
**Pull request by kevinushey**
### Intent

Addresses https://github.com/rstudio/rstudio/issues/10674.

### Approach

Tweaks to ensure pandoc and friends are solved to the right locations on Electron.

### Automated Tests

Not included; other unit tests would already cover this.

### QA Notes

Test via notes in https://github.com/rstudio/rstudio/issues/10674.

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: src/cpp/core/r_util/RTokenizer.cpp [hunk 1]
```diff
@@ -554,7 +554,7 @@ wchar_t RTokenizer::peek()
    return peek(0);
 }
 
-wchar_t RTokenizer::peek(std::size_t lookahead)
+wchar_t RTokenizer::peek(int lookahead)
```
Comment (kevinushey) [inline src/cpp/core/r_util/RTokenizer.cpp:557]: This change is tagging along to squelch a "comparing signed with unsigned" warning.

**Comment by kevinushey** [comment]
Converting to draft pending smoke test on Windows + Linux.

**Comment by kevinushey** [comment]
Confirmed that macOS + Ubuntu 20 + Windows appear to be happy with this change.

**Review by gtritchie — APPROVED** [review]
LGTM!

**Pull request merged**